styles/styles-global/root.css

```diff
@@ -32,4 +33,12 @@ body{
         "nav  foot";
 grid-template-rows: 50px 1fr 30px;
 grid-template-columns: 200px 1fr;
+}
+
+.container{
+    border: 5px solid var(--dark-grey-color) !important;
```

Comment:
Avoid using !important. instead, you can increase the specificity here.

https://stackoverflow.com/questions/27443589/how-do-i-avoid-using-important#:~:text=To%20avoid%20using%20!,%22%2C%20and%20thus%20getting%20overridden.

Reply:
looks good to merge after the changes 🚀

Reply:
Thank you for the suggestion.